pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -694,3 +694,60 @@ func TestPipelineRunsDescribe_custom_output(t *testing.T) {
 		t.Errorf("Result should be '%s' != '%s'", got, expected)
 	}
 }
+
+func TestPipelineRunDescribe_default_timeout(t *testing.T) {
```

Comment:
So the default case is covered is already in other tests, so this test can be removed.

Reply:
Removed the test and the .golden file for it.